### PR TITLE
Add relational database glossary entry

### PR DESCRIPTION
**Pull request by johnblust**
## Description & motivation
Adds relational database glossary entry to the AE glossary

### CODE REVIEW

File: website/docs/terms/relational-database.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+## Conclusion
+
+Relational databases store data in a systematic way, and support querying multiple tables together in order to generate business insights. 
+
+Often starting off with unorganized and chaotic data, analytics engineers leverage relational databases to bring structure and consistency to their data. 
+
+Relational databases also have a strong record of transactional consistency. While companies race to embrace non-relational databases in order to handle the big volume of unstructured data, most of their workloads remain transactional and analytical in nature which is why relational databases are very common. 
+
+## Additional reading
```
Comment (KiraFuruichi) [inline website/docs/terms/relational-database.md]:
```suggestion
## Further reading
```
We call this further reading in all other pages

File: website/docs/terms/relational-database.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+NoSQL databases are also typically better suited for granular real-time monitoring. On the other hand, relational databases make it easier to look at transformed and aggregated data, making them a more appropriate fit for reporting and analytics.
+
+The below table summarizes the main differences between a relational and a non-relational database:
+
+|  | Relational Database | Non-Relational Database |
+|---|---|---|
+| Data storage | Data is stored in tables. | Data is stored in document files, graph stores, key-value stores, or wide-column stores. |
+| Data format | Data is structured. | Data is mainly unstructured. |
+| Usage | Mainly used for recording transactions, data modeling, and data analysis. | Mainly used to ingest large volume real-time data streams. |
+| Data Integrity | The relationships and constraints defined ensure high data integrity.  | Non-relational databases do not guarantee data integrity. |
```
Comment (KiraFuruichi) [inline website/docs/terms/relational-database.md]:
```suggestion
| Data Integrity | The relationships and constraints defined help ensure higher data integrity.  | Non-relational databases do not guarantee data integrity. |
```

File: website/docs/terms/relational-database.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+| Scalability | Scalable at a high price tag. | Highly scalable. |
+
+## Conclusion
+
+Relational databases store data in a systematic way, and support querying multiple tables together in order to generate business insights. 
+
+Often starting off with unorganized and chaotic data, analytics engineers leverage relational databases to bring structure and consistency to their data. 
+
+Relational databases also have a strong record of transactional consistency. While companies race to embrace non-relational databases in order to handle the big volume of unstructured data, most of their workloads remain transactional and analytical in nature which is why relational databases are very common.
```
Comment (KiraFuruichi) [inline website/docs/terms/relational-database.md]:
```suggestion
Relational databases also have a strong record of transactional consistency. While some companies are racing to embrace non-relational databases in order to handle the big volume of unstructured data, most of their workloads likely remain transactional and analytical in nature which is why relational databases are very common. 
```

File: website/docs/terms/relational-database.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+Relational databases are best for structured data that can be organized into tables made up of rows and columns. Data teams rely on relational databases for storing transactional data, and also when data querying and data analysis is needed.
+
+### Transactional processing
+
+As mentioned earlier, relational databases are a great fit for transaction-oriented systems such as CRM tools, e-commerce platforms, or finance software. Companies tend to use relational databases when transactional consistency is required, as they offer a near failsafe environment for data accuracy and completion. When a transaction consists of several steps, the system treats the steps as a single transaction and assures that the operation follows an ‘all-or-nothing’ scenario, ie: the steps either all survive or all fail.
+
+### Modeling data and organizing it for analysis
+
+Relational databases support common data modeling techniques such as dimensional modeling, Data Vault, or sometimes hybrid approaches that combine different modeling techniques. Such methodologies allow teams to organize their data into useful data structures.
```
Comment (KiraFuruichi) [inline website/docs/terms/relational-database.md]:
```suggestion
Relational databases support common data modeling techniques such as <Term id = "dimensional modeling"/>, Data Vault, or sometimes hybrid approaches that combine different modeling techniques. Such methodologies allow teams to organize their data into useful data structures. 
```

**Review by KiraFuruichi — CHANGES_REQUESTED** [review]
Left some minor comments

**Comment by johnblust** [comment]
Awesome, thanks for the review @KiraFuruichi! Should be good to go now

File: website/docs/terms/relational-database.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+Relational databases are best for structured data that can be organized into tables made up of rows and columns. Data teams rely on relational databases for storing transactional data, and also when data querying and data analysis is needed.
+
+### Transactional processing
+
+As mentioned earlier, relational databases are a great fit for transaction-oriented systems such as CRM tools, e-commerce platforms, or finance software. Companies tend to use relational databases when transactional consistency is required, as they offer a near failsafe environment for data accuracy and completion. When a transaction consists of several steps, the system treats the steps as a single transaction and assures that the operation follows an ‘all-or-nothing’ scenario, ie: the steps either all survive or all fail.
+
+### Modeling data and organizing it for analysis
+
+Relational databases support common data modeling techniques such as <Term id = "dimensional modeling" />, Data Vault, or sometimes hybrid approaches that combine different modeling techniques. Such methodologies allow teams to organize their data into useful data structures.
```
Comment (KiraFuruichi) [inline website/docs/terms/relational-database.md]:
```suggestion
Relational databases support common data modeling techniques such as <Term id = "dimensional-modeling" />, Data Vault, or sometimes hybrid approaches that combine different modeling techniques. Such methodologies allow teams to organize their data into useful data structures. 
```

**Review by KiraFuruichi — CHANGES_REQUESTED** [review]
forgot hyphen

**Review by KiraFuruichi — APPROVED** [review]

**Pull request merged**